modules/dynamodb/src/main/scala/dynamodb/model.scala

```diff
@@ -98,9 +102,96 @@ object model {
 
   case class UpdateExpression(value: String)
 
+  // TODO Macro to instantiate it from static string
+  case class ExpressionAlias private (value: String)
+
+  object ExpressionAlias {
+
+    class Macros(val c: whitebox.Context) {
```

Comment:
why a `whitebox` macro here instead of a blackbox one?

Reply:
I have modified the existing macro in http4s. However, after I have read more about blackbox vs whitebox, I agree that this could be a blackbox macro instead.